packages/ember-intl/blueprints/ember-intl/index.js

```diff
@@ -26,6 +26,16 @@ module.exports = {
     };
   },
 
+  locals(options) {
+    const configInputPath = options.inAddon
+      ? 'tests/dummy/translations'
+      : 'translations';
+
+    return {
+      configInputPath,
+    };
+  },
```

Comment:
I couldn't think of a way to test this, other than publishing `ember-intl@7.0.3` and running `ember install ember-intl` in a v1 addon.